dev-docs/bidders/haxmedia.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: bidder
+title: Haxmedia
+description: Prebid Haxmedia Bidder Adapter
+biddercode: haxmedia
+usp_supported: true
+schain_supported: true
+media_types: banner, video, native
+gdpr: true
+pbjs: true
+pbs: false
+---
+
+### Prebid.Server Bid Params
```

Comment:
This should say Prebid Client Side Bid Params right?

Reply:
@ncolletti  That's a misstake, this params for Prebid.js